### PR TITLE
Add ability to import monography

### PR DESCRIPTION
**Pull request by SHtress**

### CODE REVIEW

**Review by Vikort — APPROVED** [review]

File: scripts/prepare.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ st=1
 
 build_kb=1
 build_sc_machine=1
+clone_monography=0
```
Comment (KovalM) [inline scripts/prepare.sh]:
```suggestion
clone_standard=0
```

File: scripts/prepare.sh [hunk 2]
```diff
@@ -22,6 +25,9 @@ while [ "$1" != "" ]; do
 			build_sc_machine=0
 			build_kb=0
 			;;
+		"clone_monography" )
+			clone_monography=1
```
Comment (KovalM) [inline scripts/prepare.sh]:
```suggestion
		"clone_standard" )
			clone_standard=1
```

File: scripts/prepare.sh [hunk 3]
```diff
@@ -50,7 +56,13 @@ stage "Clone projects"
 clone_project https://github.com/ShunkevichDV/sc-machine.git sc-machine 0.6.0
 clone_project https://github.com/ostis-dev/sc-web.git sc-web 0.6.0
 clone_project https://github.com/ostis-dev/ims.ostis.kb.git ims.ostis.kb 432bbaa
-
+if [ $clone_monography == 1 ]; then
+	clone_project git@github.com:semsystems/monography2020.git monography2020 tex2sc
```
Comment (KovalM) [inline scripts/prepare.sh]:
```suggestion
	clone_project git@github.com:semsystems/monography2020.git ostis-standard tex2sc
```

File: scripts/prepare.sh [hunk 4]
```diff
@@ -12,6 +12,9 @@ st=1
 
 build_kb=1
 build_sc_machine=1
+clone_monography=0
+
+monography_path="monography2020/translated_scs"
```
Comment (KovalM) [inline scripts/prepare.sh]:
```suggestion
standard_path="monography2020/translated_scs"
```

File: scripts/prepare.sh (hunk 3 above)
Comment (KovalM) [reply scripts/prepare.sh]: Move content of this "if" to separate script

File: scripts/prepare.sh [hunk 5]
```diff
@@ -50,7 +56,13 @@ stage "Clone projects"
 clone_project https://github.com/ShunkevichDV/sc-machine.git sc-machine 0.6.0
 clone_project https://github.com/ostis-dev/sc-web.git sc-web 0.6.0
 clone_project https://github.com/ostis-dev/ims.ostis.kb.git ims.ostis.kb 432bbaa
-
+if [ $clone_monography == 1 ]; then
+	clone_project git@github.com:semsystems/monography2020.git monography2020 tex2sc
+	if ! grep -q $monography_path ../repo.path ; then
+		echo "#monography" >> ../repo.path 
+		echo $monography_path >> ../repo.path
```
Comment (KovalM) [inline scripts/prepare.sh]:
```suggestion
	if ! grep -q $standard_path ../repo.path ; then
		echo "#standard" >> ../repo.path 
		echo $standard_path >> ../repo.path
```

File: scripts/prepare.sh (hunk 3 above)
Comment (KovalM) [reply scripts/prepare.sh]: tex2sc branch should be replaced by commit hash from master branch after merge pr from monography repo

**Review by KovalM — CHANGES_REQUESTED** [review]

File: scripts/clone_standart.sh [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+		git clone $1 ../$2
+		cd ../$2
+		git checkout $3
+		cd -
+	else
+		echo -en "You can update "$green"$2"$rst" manualy$rst\n"
+	fi
+}
+
+clone_project git@github.com:semsystems/monography2020.git ostis-standart tex2sc
```
Comment (KovalM) [inline scripts/clone_standart.sh]: Repo was renamed and moved to another org. Update ssh

**Review by KovalM — CHANGES_REQUESTED** [review]

**Pull request closed without merging**